Documentation~/workflow-create-polyshape.md

```diff
@@ -1,34 +1,29 @@
-# Creating a Mesh with the Poly Shape tool
+# Create a mesh with the Poly Shape tool
 
-Use the **Poly Shape** tool to define a 2-dimensional custom shape (depth and width) and then extrude a 3-dimensional shape to define its height. Like the [Shape tool](shape-tool.md), this is a modal tool that creates a new Mesh but also lets you re-activate the tool to edit the initial shape. 
+To create a custom shape, use the **Poly Shape** tool. Draw a 2D custom shape (depth and width) and then extrude that shape into a 3D mesh to define its height. 
 
-> **Note**: When you modify control points or settings, you lose any changes you made in editing the Mesh.
+![A Poly Shape is a 3D shape extruded from any 2D polygon](images/PolyShape_HeaderImage.png)
 
-![A Poly Shape is a 3-dimensional shape extruded from any 2-dimensional polygon](images/PolyShape_HeaderImage.png)
+To create a custom mesh:
 
-To define a custom Mesh:
+1. Use one of the following:
+    * In the **Scene** view > **Tools** overlay, select **Create PolyShape**.
```

Comment:
```suggestion
    * In the Scene view, in the **Tools** overlay, select **Create PolyShape**.
```

Reply:
Style guide says **Scene** should be in bold

Reply:
My bad. Disregard then.